### PR TITLE
Create Table

### PR DESCRIPTION
**Pull request by chloewlin**
@gaearon This is the screenshot of part of the table I created. Since every article gets translated to different numbers of languages, I try to keep it simple by having only three columns. 

Markdown files do not support fixed width for tables so I wasn't able to create a perfectly organized table in README.md. I also had to use line breaks. Not sure if this is best practice. 

Let me know if there is anything else I should add to it! :) 

screen shot 2019-02-10 at 9 11 15 pm

### CODE REVIEW

**Comment by WendellAdriel** [comment]
@chloewlin and @gaearon 
Should we keep a similar table with the languages and maintainers for each language?
So when new translations came in, the maintainer(s) for the language can review them before merging.

**Comment by chloewlin** [comment]
@WendellAdriel Good call! I will create a separate table for maintainers.

Originally, I want to include dates and the github handle of the current translator. But these might not be necessary. Let me know if you have any suggestions.

**Comment by WendellAdriel** [comment]
I think that a simple table like this:

| Language | Maintainer(s) |
| ----------  | --------------  |
| en | @gaearon |

Can already do the job, what do you think @chloewlin?

**Comment by chloewlin** [comment]
Yeah, I agree. It's better to keep it simple. I will add an extra table to it soon!

**Comment by chloewlin** [comment]
@WendellAdriel Done. I followed [this](https://github.com/gaearon/overreacted.io/blob/master/i18n.js) to create this table. I decided to use language rather than language code to avoid confusion.

I don't know who the other language maintainers are. Couldn't find a list in past PR either. Let me know if you know. 

screen shot 2019-02-11 at 5 53 14 am

File: README.md [hunk 1]
```diff
@@ -13,3 +13,58 @@ Add a Markdown file with the translation to the corresponding article folder. Fo
...
+|---------------------|-------------------------|
+| English             |  @gaearon               |
+| Русский             |                         |
+| Türkçe     			    |                         |
+| Español    			    |                         |
+| 한국어           	    |                         |
+| Svenska    			    |                         |
+| Italiano        	  |                         |
+| Bahasa Indonesia    |                         |
+| Português do Brasil |                         |
```
Comment (WendellAdriel) [inline README.md]:
```suggestion
| Português do Brasil |  @WendellAdriel          |
```

Just adding myself as a maintainer in the table :wink:

**Comment by WendellAdriel** [comment]
Great, using the language name for sure is better!!!
Now, about the maintainers, on #198, @gaearon has listed some on the first comment:
https://github.com/gaearon/overreacted.io/issues/198#issue-403609688

File: README.md (hunk 1 above)
Comment (chloewlin) [reply README.md]: Ok I am going to add it from my end.

Comment (chloewlin) [reply README.md]: This table looks okay in my text editor but looks so messy when I use `git diff` and here...😫 
screen shot 2019-02-11 at 6 04 29 am

Comment (WendellAdriel) [reply README.md]: Thank you so much @chloewlin!!!

Comment (WendellAdriel) [reply README.md]: Yeah, sometimes this happens with me too.
I really appreciate when tables are "organized", but it's hard to make them to be ok in text editor, console and github at the same time 😭

**Comment by chloewlin** [comment]
Everyone from that PR is added! @WendellAdriel 

screen shot 2019-02-11 at 6 27 02 am

**Comment by WendellAdriel** [comment]
Looks great to me, @chloewlin! 😉 
Let's wait on @gaearon now!!!

File: README.md (hunk 1 above)
Comment (chloewlin) [reply README.md]: Ahh. Glad to know that I am not crazy. 😊

File: README.md [hunk 2]
```diff
@@ -13,3 +13,58 @@ Add a Markdown file with the translation to the corresponding article folder. Fo
...
+| Optimized for Change                                  | Bahasa Indonesia • Deutsch • Español • <br> Français • Polski • Português do Brasil • <br> Български • Українська |       |
+| How Does setState Know What to Do?                    | Français • 日本語 • 简体中文                                           |              |
+| My Wishlist for Hot Reloading                         | Español • Français                                                   |              |
+| Why Do React Elements Have a $$typeof Property?       | Français • Português do Brasil • 日本語                               |              |
+| How Does React Tell a Class from a Function?          | Español • Français • Magyar • Slovenčina • <br>  日本語 • 繁體中文     |               |
+| Why Do We Write super(props)? | Español • Français • Italiano • Magyar • <br> Nederlands • Norsk • Português do Brasil • <br> Slovenčina •Tiếng Việt • Türkçe • srpski • <br> Čeština • فارسی • ไทย • မြန်မာဘာသာ • <br> 日本語 • 简体中文 • 繁體中文 |        |
+
+Once your translation is complete, please submit a PR and tag the maintainer(s) of your language for revivew.
+
+| Language            |      Maintainer(s)                        |
```
Comment (j-f1) [inline README.md]: Doesn’t the CODEOWNERS file handle this? It will automatically notify people for a translation for a given language.

**Comment by gaearon** [comment]
Leaving this to @j-f1 and @chloewlin to figure out the best workflow and iterate/merge

File: README.md (hunk 2 above)
Comment (chloewlin) [reply README.md]: @j-f1 Thanks for pointing this out. In this case, let me remove the table.

**Comment by gaearon** [comment]
@chloewlin What did we decide on this?

**Comment by chloewlin** [comment]
@gaearon two options: (1) use this table, constantly update it, (2) use labels in PRs to inform people what translations are being worked on currently.

I personally prefer (2) because the format of a table in markdown is very ugly. Labels are easier to manage!

**Pull request closed without merging**